monai/metrics/loss_as_metric.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) MONAI Consortium
```

Comment:
As the metric name is `LossMetric`, I would suggest to change the file name to `loss_metric.py`.
What do you think?

Thanks.